src/stream/src/task/barrier_manager.rs

```diff
@@ -691,12 +697,14 @@ impl LocalBarrierWorker {
         let root_err = self.try_find_root_failure(err).await;
         let failed_epochs = self.state.epochs_await_on_actor(actor_id).collect_vec();
         if !failed_epochs.is_empty() {
-            self.control_stream_handle
-                .reset_stream_with_err(Status::internal(format!(
-                    "failed to collect barrier. epoch: {:?}, err: {:?}",
```

Comment:
We may use `.context` from `anyhow` to add context to an error, instead of formatting the error. Besides, debug view of `Report` will contains backtrace ([doc](https://docs.rs/thiserror-ext/latest/thiserror_ext/struct.Report.html#formatting)), which is not a good practice to embed it into the _message_ again.